packages/grafana-ui/src/components/ColorPicker/ColorSwatch.tsx

```diff
@@ -20,6 +21,7 @@ export interface Props extends React.DOMAttributes<HTMLDivElement> {
 export const ColorSwatch = React.forwardRef<HTMLDivElement, Props>(
   ({ color, label, variant = ColorSwatchVariant.Small, isSelected, ...otherProps }, ref) => {
     const theme = useTheme2();
+    const { isFocusVisible, focusProps } = useFocusRing();
```

Comment:
Just curious, why do we need to use code for this when in most places we just use css (ie `&:focus-visible`)

Reply:
I think it just saves you the stress of using both `&:focus` and `&:focus-visible` on an element. Basically, useFocusRing makes focus rings to be visible only when the user is interacting with a keyboard, not with a mouse, touch, or other input methods.